ux.symfony.com/templates/components/DocsLink.html.twig

```diff
@@ -1,16 +1,15 @@
-<a
-    {{ attributes.defaults({
-        href: url,
-        rel: isExternal ? 'external noopener noreferrer',
-    }) }}
->
-    <div class="light-container p-4">
-        <p class="ubuntu-header">
-            {{ title }}
+<div {{ attributes.defaults({ class: 'DocsLink' }) }}>
+    <div class="DocsLink_content">
+        <p class="DocsLink_title ubuntu-header">
+            <a href="{{ url }}" class="DocsLink_link"
+               rel="{{ isExternal ? 'external noopened noreferrer' }}"
+            >{{ title }}</a>
             {% if isExternal %}
-                <twig:Icon name="arrow-right" style="transform: rotate(-45deg);" />
+            <twig:Icon name="arrow-right" style="transform: rotate(-45deg);" class="DocsLink_arrow" />
```

Comment:
I prefer indenting inside of Twig if statements. Not sure if there is a standard about that.

Reply:
I prefer too... but with this project my PhpStorm seems lost (example: it suggests me simple quotes instead of double one in Twig)

Fixed